pkgs/servers/monitoring/telegraf/default.nix

```diff
@@ -27,6 +27,6 @@ buildGoPackage rec {
     description = "The plugin-driven server agent for collecting & reporting metrics.";
     license = licenses.mit;
     homepage = https://www.influxdata.com/time-series-platform/telegraf/;
-    maintainers = with maintainers; [ mic92 roblabla ];
+    maintainers = with maintainers; [ mic92 roblabla foxit64 ];
```

Comment:
also add yourself to maintainers.nix

Reply:
Done! https://github.com/NixOS/nixpkgs/pull/78709